sites/all/modules/reol_review/README.md

```diff
@@ -0,0 +1,10 @@
+# Reol review
+
+## Check availability of reviews
+
+Run the `'reol_review_check` to check availability of reviews, i.e. check the
+http code when `GET`'ing the actual review.
+
+```sh
+drush php-eval 'reol_review_check()'
```

Comment:
Why not make an drush file an make it a real drush command. 20 linjes of code?

Reply:
Stuff moved to `drush` command.